README.md

```diff
@@ -66,6 +68,31 @@
 | Internet Explorer           | 10 & 11 |
 | Safari & Mobile Safari      | Current |
 
+## Internationalization(I18n)
+
+1. Please follow [Base Getting Started](packages/terra-base/README.md#getting-started) to install `Base`, and consume it with `locale` props.
+2. Install and config `terra-i18n-plugin`:
+    - Follow [terra-i18n-plugin Getting Started](packages/terra-i18n-plugin#getting-started) to install `terra-i18n-plugin` to aggregate translations in `node_modules`.
+    - Follow [terra-i18n-plugin Wiki Guide](https://github.com/cerner/terra-core/wiki/Terra-i18n-plugin-Guide) to config i18n plugin.
+3. Insatll and config `react-intl` 
+    - Install it `npm install --save react-intl`.
+    - Add alias webpack config to avoid importing duplicate `react-intl`.
+        ```
+        resolve: {
+          extensions: ['.js', '.jsx'],
+          alias: {
+            'react-intl': path.resolve(__dirname, 'node_modules/react-intl'),
+          },
+        },
+        ```
+4. Based on your strategy to set locale by backend function and provide `customMessages` prop to `Base` by backend translations.
```

Comment:
I think the Based on your strategy part of the sentence is confusing. I would just say

    Provide values for locale and customMessages.

Perhaps we should mention that it's recommended to set the lang attribute on the html node as well?